new_lrt.ttl

```diff
@@ -373,6 +381,54 @@ Beispiele:, Motivbild, Muster, Ornament, Icon, Muster"""@de ;
     skos:prefLabel "Kurs"@de ;
     skos:scopeNote "Monate"@de .
 
+<8d5195dd-2e48-44d4-a9c1-6bccbf85ec96> a skos:Concept ;
```

Comment:
@lummerland can we add related mappings to the elm list? (May add them as a new SKOS list as well)
https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://data.europa.eu/snb/learning-opportunity/25831c2

Reply:
Relations were added, but I did not fetch the ELM list in SKOS. Not sure how useful this would be.